apps/content-publishing-worker/src/batch_announcer/batch.announcer.ts

```diff
@@ -73,13 +73,13 @@ export class BatchAnnouncer {
     // Get previously uploaded file from IPFS
     this.logger.log(`Getting info from IPFS for ${batch.cid}`);
     try {
-      const size = await this.ipfsService.contentLengthInLocalGateway(batch.cid);
-      this.logger.debug(`Got info from IPFS: size=${size}`);
+      const { cid, size } = await this.ipfsService.getInfoFromLocalNode(batch.cid);
+      this.logger.debug(`Got info from IPFS: cid=${cid}, size=${size}`);
 
       const response = {
         id: batch.cid,
         schemaId: batch.schemaId,
-        data: { cid: batch.cid, payloadLength: size },
+        data: { cid: cid.toV1().toString(), payloadLength: size },
```

Comment:
Informational only: Are we only ever going to support v1 CIDs?  we were discussing CIDv2 at some point.

Reply:
The issue is, CIDv1 is deterministic based solely on the hashed file contents. CIDv2 is additionally dependent on the chunking parameters with which the file was uploaded/stored on IPFS. So: we can pre-compute a CIDv1 based on the file contents, but any CIDv2 we attempt to pre-compute will not necessarily match the CIDv2 that IPFS generates for content addressing.

Thus, for our purposes, it's always CIDv1.